### DISCUSSION TITLE
比例描述不统一

### DISCUSSION THREAD

**Discussion by YangFong** [alovajs/alova#156] [General]
虽说「30%+」还有「95% 以上」这些描述是包括了更高数值的，只是这样看起来还是怪怪的：

![image](https://github.com/alovajs/alova/assets/70502828/87d831ae-8321-473e-8e34-7cfa799775e7)

![image](https://github.com/alovajs/alova/assets/70502828/89109fa2-29c5-4e98-a66c-9734d55eab35)

**Comment by JOU-amjs** [comment]
明察秋毫的眼睛，已更正
